benches/benches/splitsquaremap/init.rs

```diff
@@ -25,7 +25,7 @@ pub fn criterion_benchmark(c: &mut Criterion) {
         group.bench_with_input(BenchmarkId::new("init", ""), &n_square, |b, n_square| {
             b.iter(|| {
                 let mut map: CMap2<FloatType> =
-                    GridBuilder::split_unit_squares(*n_square).build2().unwrap();
+                    CMapBuilder::unit_split_grid(*n_square).build().unwrap();
```

Comment:
`unit_split_grid`?
The name is not very obvious.

Reply:
Yeah, I'm not satisfied either. Is `unit_triangles` better?